<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deleteitem_seller</class>
 <widget class="QDialog" name="deleteitem_seller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_namedeleteseller">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_deleteproductid">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_namedeleteseller">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_deleteproductid">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_submitdeleteitem">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
